ipn/ipnserver/server.go

```diff
@@ -696,34 +695,15 @@ func StateStore(path string, logf logger.Logf) (ipn.StateStore, error) {
 	if path == "" {
```

Comment:
Line 685: isn't "file:" also a special case now? It's optional? Only used for Windows?

Reply:
Removed those from here and pointed to ipn.Store.